### PR TITLE
Adds Aim Docker Space Docs

### PR DESCRIPTION
**Pull request by grigoryan-davit**
First version of Aim docs. Feedback more than welcome.
TODO: replace one of the screenshots with the appropriate one when the Aim Docker template is ready.

### CODE REVIEW

File: docs/hub/spaces-sdks-docker-aim.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
+# Aim on Spaces
+
+**Aim** is an easy-to-use & supercharged open-source experiment tracker. Aim logs your training runs, enables a beautiful UI to compare them and an API to query them programmatically.
```
Comment (osanseviero) [inline docs/hub/spaces-sdks-docker-aim.md:3]:
```suggestion
**Aim** is an easy-to-use & supercharged open-source experiment tracker. Aim logs your training runs and enables a beautiful UI to compare them and an API to query them programmatically.
```

File: docs/hub/spaces-sdks-docker-aim.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+Check out the [Aim docs](https://aimstack.readthedocs.io/en/latest/) to learn more about Aim. 
+If you have an idea for a new feature or have noticed a bug, feel free to [open a feature request or report a bug](https://github.com/aimhubio/aim/issues/new/choose). 
+
+In the following sections, you'll learn how to deploy Aim on the Hugging Face Hub Spaces and explore your training runs directly from the Hub. 
+
+## Deploy Aim on Spaces
+
+You can deploy Aim on Spaces with just a few clicks:
+
+<a  href="https://huggingface.co/new-space?template=argilla/argilla-template-space">
```
Comment (osanseviero) [inline docs/hub/spaces-sdks-docker-aim.md:15]: TODO: update

File: docs/hub/spaces-sdks-docker-aim.md [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+    <img src="https://huggingface.co/datasets/huggingface/badges/raw/main/deploy-to-spaces-lg.svg" />
+</a>
+
+All you need to do is:
+1. Define the owner (your personal account or organization)
+2. Name your space
+3. Choose the **Aim** Docker template
```
Comment (osanseviero) [inline docs/hub/spaces-sdks-docker-aim.md:22]: This will be done automatically with the right link

File: docs/hub/spaces-sdks-docker-aim.md [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+
+You can deploy Aim on Spaces with just a few clicks:
+
+<a  href="https://huggingface.co/new-space?template=argilla/argilla-template-space">
+    <img src="https://huggingface.co/datasets/huggingface/badges/raw/main/deploy-to-spaces-lg.svg" />
+</a>
+
+All you need to do is:
+1. Define the owner (your personal account or organization)
+2. Name your space
```
Comment (osanseviero) [inline docs/hub/spaces-sdks-docker-aim.md:21]:
```suggestion
2. Name your Space
```

File: docs/hub/spaces-sdks-docker-aim.md [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+3. Choose the **Aim** Docker template
+4. Set the visibility
+
+Once you have created the space, you'll see the `Building` status and once it becomes `Running` your space is ready to go!
```
Comment (osanseviero) [inline docs/hub/spaces-sdks-docker-aim.md:25]:
```suggestion
Once you have created the Space, you'll see the `Building` status, and once it becomes `Running,` your Space is ready to go!
```

File: docs/hub/spaces-sdks-docker-aim.md [hunk 6]
```diff
@@ -0,0 +1,114 @@
...
+
+### TODO: change screenshot once the Aim template is available
+![Creating an Aim Space ](https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/spaces-livebook-new-space.png)
+
+Now, when you navigate to the **App** section of your space, you'll be able to access the Aim UI.
```
Comment (osanseviero) [inline docs/hub/spaces-sdks-docker-aim.md:30]:
```suggestion
Now, when you navigate to your Space's **App** section, you can access the Aim UI.
```

File: docs/hub/spaces-sdks-docker-aim.md [hunk 7]
```diff
@@ -0,0 +1,114 @@
...
+
+## Compare your experiments with Aim on Spaces
+Let's use a quick example of a PyTorch CNN trained on MNIST to demonstrate end-to-end Aim on Spaces deployment. 
+The full example in the [Aim repo examples folder](https://github.com/aimhubio/aim/blob/main/examples/pytorch_track.py).
```
Comment (osanseviero) [inline docs/hub/spaces-sdks-docker-aim.md:34]:
```suggestion
The full example is in the [Aim repo examples folder](https://github.com/aimhubio/aim/blob/main/examples/pytorch_track.py).
```

File: docs/hub/spaces-sdks-docker-aim.md [hunk 8]
```diff
@@ -0,0 +1,114 @@
...
+...
+items = {'accuracy': acc, 'loss': loss}
+aim_run.track(items, epoch=epoch, context={'subset': 'train'})
+
+# Track weights and gradients distributions
+track_params_dists(model, aim_run)
+track_gradients_dists(model, aim_run)
+```
+
+The experiments tracked bdy Aim are stored in the `.aim` folder. **To display the logs with the Aim UI in your HF space, you need to compress the `.aim` folder to a `tar.gz` file and upload it to your Space either using `git` or the Files and Versions sections of your space.**
```
Comment (osanseviero) [inline docs/hub/spaces-sdks-docker-aim.md:51]:
```suggestion
The experiments tracked by Aim are stored in the `.aim` folder. **To display the logs with the Aim UI in your Space, you need to compress the `.aim` folder to a `tar.gz` file and upload it to your Space using `git` or the Files and Versions sections of your Space.** 
```

File: docs/hub/spaces-sdks-docker-aim.md [hunk 9]
```diff
@@ -0,0 +1,114 @@
...
+
+![image](https://user-images.githubusercontent.com/67782184/224995427-db0d8a2b-1d48-46d1-b519-78a2d4d5d0aa.png)
+
+Check out a [text2speech example ](https://huggingface.co/spaces/aimstack/text2speech/tree/main) on HF Hub Spaces.
+
+**_Note_** that if your logs are in tensorboard format, you can easily convert [them to Aim with one command](https://aimstack.readthedocs.io/en/latest/quick_start/convert_data.html#show-tensorboard-logs-in-aim) and use the many advanced and high-performant training run comparison features available.
+
+## Track your Transformers runs with Aim
+
+As we saw in the previous section, tracking Pytorch experiments with Aim and displaying the results on Spaces is straightforward. To make your life even easier, Aim has out-of-the-box integration with Transformers!
```
Comment (osanseviero) [inline docs/hub/spaces-sdks-docker-aim.md:83]:
```suggestion
As we saw in the previous section, tracking PyTorch experiments with Aim and displaying the results on Spaces is straightforward. To make your life even easier, Aim has out-of-the-box integration with Transformers! 
```

File: docs/hub/spaces-sdks-docker-aim.md [hunk 10]
```diff
@@ -0,0 +1,114 @@
...
+Filter your runs using Aim’s Pythonic search. You can write pythonic [queries against](https://aimstack.readthedocs.io/en/latest/using/search.html) EVERYTHING you have tracked - metrics, hyperparams etc.
+In the screenshot below, we query all the `accuracy` metrics where the last tracked value is `≥ 90`.
+
+![image](https://user-images.githubusercontent.com/67782184/224995427-db0d8a2b-1d48-46d1-b519-78a2d4d5d0aa.png)
+
+Check out a [text2speech example ](https://huggingface.co/spaces/aimstack/text2speech/tree/main) on HF Hub Spaces.
+
+**_Note_** that if your logs are in tensorboard format, you can easily convert [them to Aim with one command](https://aimstack.readthedocs.io/en/latest/quick_start/convert_data.html#show-tensorboard-logs-in-aim) and use the many advanced and high-performant training run comparison features available.
+
+## Track your Transformers runs with Aim
```
Comment (osanseviero) [inline docs/hub/spaces-sdks-docker-aim.md:81]: I'm not sure if this should be in the documentation of the Hub, as that's mostly about using the Space. This could maybe be in the transformers docs but also not sure as this is not a in-library callback

**Review by osanseviero — COMMENTED** [review]
Very cool! Thanks a lot! :fire:

File: docs/hub/spaces-sdks-docker-aim.md (hunk 10 above)
Comment (SGevorg) [reply docs/hub/spaces-sdks-docker-aim.md:81]: Yea, our thinking was to make sure there is extra context. We can just link to our Hugging Face integration docs in that case?

Comment (osanseviero) [reply docs/hub/spaces-sdks-docker-aim.md:81]: I think so. Or maybe just have a 2-3 sentence description here and then link to the transformers docs. That will also be easier to maintain

File: docs/hub/spaces-sdks-docker-aim.md [hunk 11]
```diff
@@ -0,0 +1,114 @@
...
+Check out the [Aim docs](https://aimstack.readthedocs.io/en/latest/) to learn more about Aim. 
+If you have an idea for a new feature or have noticed a bug, feel free to [open a feature request or report a bug](https://github.com/aimhubio/aim/issues/new/choose). 
+
+In the following sections, you'll learn how to deploy Aim on the Hugging Face Hub Spaces and explore your training runs directly from the Hub. 
+
+## Deploy Aim on Spaces
+
+You can deploy Aim on Spaces with just a few clicks:
+
+<a  href="https://huggingface.co/new-space?template=argilla/argilla-template-space">
```
Comment (julien-c) [inline docs/hub/spaces-sdks-docker-aim.md:15]:
```suggestion
<a  href="https://huggingface.co/new-space?template=aimstack/text2speech">
```

🔥

**Review by julien-c — COMMENTED** [review]
looks really cool!

File: docs/hub/spaces-sdks-docker-aim.md [hunk 12]
```diff
@@ -0,0 +1,73 @@
...
+
+Filter your runs using Aim’s Pythonic search. You can write pythonic [queries against](https://aimstack.readthedocs.io/en/latest/using/search.html) EVERYTHING you have tracked - metrics, hyperparams etc. Check out some [examples](https://huggingface.co/aimstack) on HF Hub Spaces.
+
+<Tip>
+Note that if your logs are in TensorBoard format, you can easily convert <a href="https://aimstack.readthedocs.io/en/latest/quick_start/convert_data.html#show-tensorboard-logs-in-aim">them to Aim with one command</a> and use the many advanced and high-performant training run comparison features available.
+</Tip>
+
+## More on HF Spaces
+
+- [HF Docker spaces](https://github.com/huggingface/hub-docs/blob/main/docs/hub/spaces-sdks-docker.md)
```
Comment (osanseviero) [inline docs/hub/spaces-sdks-docker-aim.md:66]:
```suggestion
- [HF Docker spaces](https://huggingface.co/docs/hub/spaces-sdks-docker)
```

File: docs/hub/spaces-sdks-docker-aim.md [hunk 13]
```diff
@@ -0,0 +1,73 @@
...
+- [HF Docker space examples](https://github.com/huggingface/hub-docs/blob/main/docs/hub/spaces-sdks-docker.md)
```
Comment (osanseviero) [inline docs/hub/spaces-sdks-docker-aim.md:67]:
```suggestion
- [HF Docker space examples](https://huggingface.co/docs/hub/spaces-sdks-docker-examples)
```

**Review by osanseviero — APPROVED** [review]
Thanks! :rocket:
